probe-rs/src/debug/mod.rs

```diff
@@ -161,15 +187,28 @@ impl<'a, 'b> Iterator for StackFrameIterator<'a, 'b> {
         let pc = match self.pc {
             Some(pc) => pc,
             None => {
+                debug!("Unable to determine next frame, program counter is zero");
                 return None;
             }
         };
 
-        let unwind_info = self
-            .debug_info
-            .frame_section
-            .unwind_info_for_address(&bases, &mut ctx, pc, gimli::DebugFrame::cie_from_offset)
-            .unwrap();
+        let unwind_info = self.debug_info.frame_section.unwind_info_for_address(
+            &bases,
+            &mut ctx,
+            pc,
+            gimli::DebugFrame::cie_from_offset,
+        );
+
+        let unwind_info = match unwind_info {
+            Ok(uw) => uw,
+            Err(e) => {
+                info!(
```

Comment:
Shouldn't this be an error or is this an info because it's not an error per se but rather just missing info?

Reply:
The stack frame unwinding is only "best-effort" anyway, so I think just returning `None` should be okay. 

I'm not sure if this occurs because of missing debug info or because of an error in handling it, but for usage in a debugger an error doesn't really help, and for debugging the log should be enough.

Reply:
oh, i just meant the log output :) the code flow itself seems ok to me!